modules/prometheus/src/main/scala/com/evolutiongaming/smetrics/Prometheus.scala

```diff
@@ -0,0 +1,244 @@
+package com.evolutiongaming.smetrics
+
+import java.io.StringWriter
+
+import cats.effect.{Resource, Sync}
+import cats.implicits._
+import io.prometheus.client.Collector
+import io.prometheus.client.exporter.common.TextFormat
+import io.prometheus.{client => P}
+
+trait Prometheus[F[_]] {
+
+  val registry: CollectorRegistry[F]
```

Comment:
please define these always as `def` in basic interface, however implement via `val` in the implementation